Clone page

Maryjanee opened the pull request:
Completed Youtube clone project

Maryjanee committed:
add images and starter files

Maryjanee committed:
add nav and nav styles

Maryjanee committed:
add more navbar styles and video section

Commit:
add styles to main container

Commit:
add title container

Maryjanee committed:
add views and video description

Commit:
fix git error

Commit:
fix git merge conflict

Commit:
add styles to subscribe button

Commit:
add the discription video tag

Maryjanee committed:
add more styles to description

Maryjanee committed:
add aside title

Commit:
add the thumbnail images

Maryjanee committed:
add aside styles

Commit:
add more thumbnail container

Maryjanee committed:
add more styles to thumbnail and changed icons from png to svg

Commit:
add more images on png format

Maryjanee committed:
add new icons to page

Maryjanee committed:
update read me file

sumancrest0001 reviewed (commented):
**Status: Changes required
Hello TEAM :wave:** 
You have done a nice job to create an almost similar layout, the readme file is also looking good. But there are some minor issues that need to be addressed before moving ahead. let's fix them.
[Required] The YouTube button on the left should align with other contents. Currently, they are not aligned properly. :point_down: 
![Screenshot from 2020-06-04 07-41-49](https://user-images.githubusercontent.com/37747619/83706593-9a475680-a637-11ea-9203-a380969ee13f.png)
It should be :point_down: 
![Screenshot from 2020-06-04 07-47-37](https://user-images.githubusercontent.com/37747619/83706626-b3e89e00-a637-11ea-813e-91976b17ed0a.png)

[Optional] You can change the logo also to give it similar looks.

sumancrest0001 commented on index.html:
```diff
@@ -0,0 +1,162 @@
+                  </li>
+                  <li class="icons">
+                    <img src="images/dislike.svg" alt="dislike" />18
+                  </li>
+                  <li class="icons">
+                    <img src="images/share.svg" alt="share" /> share
+                  </li>
+                </ul>
```
[Required] Share icon should be a button but it is an item only in your layout. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) of the requirement.

[Required] Thumbnail images of other videos are not showing.I am using [githack](https://raw.githack.com/) to check the project.  Have a look :point_down: 
![Screenshot from 2020-06-04 07-37-30](https://user-images.githubusercontent.com/37747619/83707912-ddef8f80-a63a-11ea-9b6b-b6829cf5cd23.png)

sumancrest0001 reviewed (changes requested):
**Kindly make the requested changes and request another review.**

If you have any queries regarding the review or need support, message me on slack **Suman Shrestha**. I am happy to help you.
**Happy coding !!!**

Maryjanee committed:
add new images and made requested changes

Commit:
add the subscribe section

Maryjanee committed:
fix icon size and style subscribe button

Commit:
add the readme live demo

rahalrazika commented:
All changes requested have been done and the readme file has been updated with a live demo of the project.

Maryjanee committed:
update read me file

elmejdki commented:
### Pull request under review by Zakariae El Mejdki.

elmejdki commented on README.md:
```diff
@@ -1 +1,40 @@
+
+👤 **Razika**
+
+- Github: [@rahalrazika](https://github.com/rahalrazika)
```
[OPTIONAL] if it is possible add also your Linkedin and Twitter profile links here.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* Basic resets */
```
kindly remove all the comments from all the files:
![image](https://user-images.githubusercontent.com/34653764/83825179-7b48d300-a6d0-11ea-852c-3d54ac4bc2ff.png)

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  padding-left: 3.2rem;
+}
+
+/* aside styles */
```
remove this comment also.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  justify-content: space-between;
+}
+
+/* Video-container */
```
also this one.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  margin-right: 0;
+}
+
+/*Main Styles */
```
and this one.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  width: 8rem;
+}
+
+/* Form Styles */
```
another comment to be deleted.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  font-family: "Roboto", sans-serif;
+}
+
+/* General styles */
```
remove this comment as suggested above.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,194 @@
+  cursor: pointer;
+}
+
+/* Navigation Styles */
```
and this one.

elmejdki reviewed (changes requested):
# Great job so far team :woman_technologist: :tophat: 
### status: require changes

- Add a meaningful title and summary on your pull request, you can use on the title *Completed Youtube clone project*, and on the summary try to add some bullet point of what you have accomplished, 

- use this link [HTLM validator](https://validator.w3.org/#validate_by_uri) to validate your HTML code, while validating it I have found the following errors:

![image](https://user-images.githubusercontent.com/34653764/83825622-b992c200-a6d1-11ea-8786-6fd6f7206778.png)
- To fix this error please remove any HTML styling attributes and use CSS instead. 

![image](https://user-images.githubusercontent.com/34653764/83825722-155d4b00-a6d2-11ea-8a79-5b9129c9c9ed.png)
- to fix this one also you need to add a heading and then use a display none in your CSS, to make it invisible.

### please make the required changes before requesting another pull request
_if you have any question be free to contact me on slack you will find me as **Zakariae El Mejdki**_

Maryjanee committed:
fix issues in html file

Maryjanee committed:
update readme file and css file

Maryjanee commented:
- Added navigation bar
- Embedded Video and videos on webpage
- Added sidebar with recommended images.
- Removed the comments in the css file
- Validated index File with HTML validator and fixed all resulting issues.

HADDADSOHAIB commented on index.html:
```diff
@@ -0,0 +1,173 @@
+        </div>
+        <div class="thumbnail-container">
+          <div id="first-video" class="thumbnail d-flex">
+            <img src="images/1.png" alt="thumbnail_1" />
```
I see that the name of your file in the image folder is 1.PNG, that could be the issue why the images are not showing

HADDADSOHAIB reviewed (changes requested):
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @Maryjanee and @rahalrazika     :handshake: 

### You did wonderful job :clap:  your project is looking good :medal_sports: and look like the original. 

### You implemented the changes required by the previous TSE, good job, however I found some issues while checking your project latest version using https://raw.githack.com/, (the live demo provided in the readme is ok), there are some changes to get the project approved :point_down:
- the thumbnails are not showing, check the comments for suggesting.
![Screenshot from 2020-06-05 13-06-44](https://user-images.githubusercontent.com/26064042/83874875-13c17080-a72e-11ea-96e5-15eb4e99a27a.png)
 ### Please fix these issues and asked for another code review.

Maryjanee committed:
fix issue with images

Maryjanee committed:
fix issue with images

BrittanyBlake commented:
### Pull Request Under Review by Brittany Blake (TSE)

BrittanyBlake reviewed (approved):
### Status: Approved ✅
****
### Hi @Maryjanee and @rahalrazika 👋 👋 
You have done a great job implementing the changes requested by the previous TSEs! 👍 
### Congratulations 🏅 👏 
Go ahead and merge this pull request and move on to the next project 💪 

HAPPY CODING! 👩🏾‍💻
